assets/templates/template/src/game.ts

```diff
@@ -6,6 +6,18 @@ import {
     CannonJSPlugin
 } from 'babylonjs';
 
+// In case you are using materials from the materials library. Import the library.
+import 'babylonjs-materials';
+
+// In case you are using procedural textures. Import the library.
+import 'babylonjs-procedural-textures';
+
+// In case you have exported your scene in a format different form .babylon. Import the library.
```

Comment:
« ... different FROM .babylon »
Sorry I badly wrote that word on the forum ^^

Reply:
@julien-moreau 
ah right. I am still testing this. I tried updating system.js and it behaved oddly, I'm fishing around for a good version.

I will change these comments to be more clear, this is till a work in progress, I'll mark that